File: Programmers/src/step1/RectangularStar.java
package step1;

import java.util.Scanner;

public class RectangularStar {

	/**
	* ABOUT
	*
	* DATE: 2022-02-12
	* AUTHOR: dms873
	*
	* URL: https://programmers.co.kr/learn/courses/30/lessons/12969?language=java
	*
	* COMMENT:
	* for 문을 이용하여 별찍기
	**/
	
	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt();
		int b = sc.nextInt();
		
		for(int i = 0; i < b; i++) {
			for(int j = 1; j <= a; j++) {				
				System.out.print("*");
			}
			System.out.println();				
		}
		sc.close();
		
	}
}

File: Programmers/src/step1/RectangularStar.js
/**
	* ABOUT
	*
	* DATE: 2022-02-14
	* AUTHOR: dms873
	*
	* URL: https://programmers.co.kr/learn/courses/30/lessons/12969?language=javascript
	*
	* COMMENT:
	* for 문을 이용하여 별찍기
	**/

process.stdin.setEncoding('utf8');
process.stdin.on('data', data => {
    const n = data.split(" ");
    const a = Number(n[0]), b = Number(n[1]);
    for(let i = 0; i < b; i++) {
    console.log('*'.repeat(a));    
    }

});
